packages/trpc/server/routers/viewer/teams/skipTeamTrials.handler.ts

```diff
@@ -29,18 +29,19 @@ export const skipTeamTrialsHandler = async ({ ctx }: SkipTeamTrialsOptions) => {
       },
     });
 
-    const ownedTeams = await prisma.team.findMany({
+    const memberships = await prisma.membership.findMany({
       where: {
-        members: {
-          some: {
-            userId: ctx.user.id,
-            accepted: true,
-            role: "OWNER",
-          },
-        },
+        userId: ctx.user.id,
+        accepted: true,
+        role: "OWNER",
+      },
+      select: {
+        team: true,
       },
     });
 
+    const ownedTeams = memberships.map((membership) => membership.team);
```

Comment:
NIT: Personally I'm not a huge fan of iterating over this array here and on line 45.